cli/src/config.rs

```diff
@@ -71,6 +73,7 @@ impl Default for Configuration {
             public_key,
             private_key,
             disable_panic_terminal_colors: bool::default(),
+            enable_shutdown_on_panic: Default::default(),
```

Comment:
Why not just `false`? It's both shorter and more direct.

Reply:
Less refactoring in case of type changing. But I'll change it to `false`, thanks.